examples/repaire/main.go

```diff
@@ -28,10 +28,10 @@ func repaireLas(fileName string) {
 	las.FileName = fileName
 	fmt.Printf("file: '%s'", fileName)
 	n, err := las.Open(fileName) // считываем файл
-	fmt.Printf(" readed\n")
+	fmt.Printf(" read\n")
```

Comment:
The spelling 'read' is used for both present-tense and past-tense.  It sounds like 'reed' when used in present-tense.  It sounds like 'red' when used in past-tense.